torchbenchmark/models/demucs/__init__.py

```diff
@@ -34,7 +38,7 @@ def forward(self, streams):
 
 class Model(BenchmarkModel):
     task = OTHER.OTHER_TASKS
-    def __init__(self, device=None, jit=False):
+    def __init__(self, device: Optional[str]=None, jit: bool=True) -> None:
```

Comment:
Let's set `jit` default value to be `False` in the Model constructor as all the other benchmarks do. We will create Model instances for `jit=True` for jit-runs.